<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogDialogClass</class>
 <widget class="QDialog" name="LogDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/img/dbmaster.png</normaloff>:/img/dbmaster.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>F&amp;ilter</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Message</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeft|AlignTop</set>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="clearButton">
     <property name="text">
      <string/>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>LogDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
